src/Native/gc/sample/gcenv.ee.cpp

```diff
@@ -15,6 +15,8 @@ int32_t g_TrapReturningThreads;
 
 EEConfig * g_pConfig;
 
+gc_alloc_context g_global_alloc_context;
```

Comment:
FWIW, this is not needed. Deleting in https://github.com/dotnet/coreclr/pull/24204